README.md

```diff
@@ -1,73 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Leaderboard App
 
+This project send data to the leaderboad api and get it from the same api to display the score when the person adds it
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
-## Live Demo (if available)
+## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Kindly ensure to remove this live demo link if you have not yet hosted a live demo. This will make your ReadMe look great!